Sources/Thumbprint/Presentations/Presentation.swift

```diff
@@ -1,5 +1,6 @@
 import UIKit
 
+@MainActor
 public enum Presentation {
```

Comment:
I'm confused, you originally were trying to make `PartialSheetPresentation` be Sendable, which is a different object ? 

I think the enum, should just be Sendable, and you can mark `PartialSheetPresentation` as @MainActor right ?

Reply:
if i don't make it mainActor, I will get this error when setting PartialSheetPresentation to @mainactor
<img width="941" alt="Screenshot 2024-07-10 at 10 40 04 AM" src="https://github.com/thumbtack/thumbprint-ios/assets/113396903/60064ef7-a6a0-47ce-860a-6a2ab1e358c2">

Reply:
i missed that you did change PartialSheetPresentation as well.